src/middleware/auth.ts

```diff
@@ -0,0 +1,25 @@
+import { NextResponse } from "next/server";
+import type { NextRequest } from "next/server";
+
+const username = process.env.BASIC_AUTH_USERNAME || "admin";
```

Comment:
no default username and password

Reply:
Entfernt und wirft jetzt Fehler wenn nicht gesetzt